posts/TechYesWeCan_Pt1.md

```diff
@@ -0,0 +1,115 @@
+---
+authors:
+- Leigha Dellinger, Sarah Narum, Rita Barlow, Trish Brumett, Erin Geoghan, Ramya Shetty
+tags:
+- Women In Technology
+- RVATech
+date: 2018-10-31T12:21:50.000Z
+title: "Tech Yes We Can! Version 1.1"
+image: 
+---
+
+# 'Tech Yes She Can!’ Version 1.1
```

Comment:
Remove this title, this is redundant with the title in the meta information

Reply:
I thought it was nice formatting, however I have removed it!